QualifiedTeachersApi/src/QualifiedTeachersApi/Services/GetAnIdentity/WebHooks/GetAnIdentityEndpoints.cs

```diff
@@ -78,8 +63,24 @@ public static IEndpointConventionBuilder MapIdentityEndpoints(this IEndpointRout
 
                     await mediator.Send(request);
                 }
+                else if (notification.Message is UserCreatedMessage)
+                {
+                    var userCreatedMessage = (UserCreatedMessage)notification.Message;
+                    var request = new UserCreatedRequest()
+                    {
+                        UserId = userCreatedMessage.User.UserId,
+                        Trn = userCreatedMessage.User.Trn,
+                        EmailAddress = userCreatedMessage.User.EmailAddress,
+                        MobileNumber = userCreatedMessage.User.MobileNumber,
+                        UpdateTimeUtc = notification.TimeUtc
+                    };
+
+                    await mediator.Send(request);
+                }
 
                 return Results.NoContent();
             });
     }
+
+    static JsonSerializerOptions SerializerOptions { get; } = new JsonSerializerOptions(JsonSerializerDefaults.Web).AddConverters();
```

Comment:
Nit: fields are normally at the top of the class
```suggestion
    private static JsonSerializerOptions SerializerOptions { get; } = new JsonSerializerOptions(JsonSerializerDefaults.Web).AddConverters();
```